src/Combinatorics/Graphs/functions.jl

```diff
@@ -717,6 +717,10 @@ julia> g = complete_graph(4);
 
 julia> automorphism_group(g)
 Permutation group of degree 4
+with 3 generators
```

Comment:
Would it be better to have this all in one line?
```suggestion
Permutation group of degree 4 with 3 generators
```

Reply:
Maybe o.k. for permutation groups, but for more nested constructs it will be easier to have multiple lines 
e.g. the following seem bad to me:
`automorphism group of permutation group of degree 4 with 3 generators with 4 generators`
or even worse, when it is not clear at all where the generators belong:
`automorphism group of permutation group of degree 4 with 4 generators`

The following seems more structured to me:
```
automorphism group 
  of permutation group of degree 4 with 3 generators 
with 4 generators`
  (1,2,3) 
  (4,5,6)
```